api/client/client.go

```diff
@@ -236,6 +236,12 @@ func connect(ctx context.Context, cfg Config) (*Client, error) {
 						sshConfig: sshConfig,
 						addr:      addr,
 					})
+					syncConnect(ctx, tlsRoutingConnect, connectParams{
```

Comment:
What about something like?

```
for _, cf := range []connectFunc{proxyConnect, tunnelConnect, tlsRoutingConnect}{
    syncConnect(ctx, cf, connectParams{})
}
```

I would also update or remove the comment on line 219 as it's now out of date.

Reply:
Yes, It look cleaner. Updated, thanks.